homeassistant/components/motionmount/select.py

```diff
@@ -25,6 +25,7 @@ class MotionMountPresets(MotionMountEntity, SelectEntity):
 
     _attr_translation_key = "motionmount_preset"
     _attr_current_option: str | None = None
+    _presets: list[motionmount.Preset] = []
```

Comment:
What happens if there is more than one motionmount in use?

Using a class variable here would appear to share a common list of presets between instances. Is that what you want?

Reply:
That was definitely not what I want.
I also removed the `_attr_current_option`, as that one seems the be a class var as well if defined this way.

Reply:
`_attr_current_option` is a special case, because your class is inheriting from `Entity`, and makes use of `homeassistant.helpers.entity.CachedProperties`.

`_attr_xxxx` is a special name that uses what you supply as a class variable and creates getter and setter functions automatically behind the scenes.

So that particular class variable is not shared between instances even though it looks like it is.

I suggest `_presets` should be moved to `self._presets` within the `__init__` function, but the others can stay where they are.

If I understand correctly, using
```
    _attr_current_option: str | None = None
```
...only has the effect of setting a default, since `_attr_current_option: str | None` is already present in the parent `SelectEntity` class. This is ok to do but you probably don't need the type hint, so just `_attr_current_option = None` would be enough.
--
Happy for someone to correct me on this if I have not understood correctly or not explained it well.

Reply:
@RJPoelstra if you are happy that `_attr_current_option` no longer has the default value of `None`, then I think the PR is good to go.

ad12a74 removed the default assignment.

It could be that the library defaults the value to `None` anyway, but I am not sure and you may want to check this.

Reply:
Hi, The documentation states that `None` is the default, so I think it's all good this way.